plugins/plugin-java-debugger/che-plugin-java-debugger-server/src/main/java/org/eclipse/che/plugin/jdb/server/JavaDebugger.java

```diff
@@ -459,72 +459,63 @@ public StackFrameDumpDto dumpStackFrame() throws DebuggerException {
    */
   @Override
   public SimpleValue getValue(VariablePath variablePath) throws DebuggerException {
-    List<String> path = variablePath.getPath();
-    if (path.size() == 0) {
-      throw new IllegalArgumentException("Path to value may not be empty. ");
+    lock.lock();
+    try {
+      return getValue(variablePath, getCurrentThread().uniqueID(), 0);
+    } finally {
+      lock.unlock();
     }
-    JdiVariable variable;
+  }
+
+  @Override
+  public SimpleValue getValue(VariablePath variablePath, long threadId, int frameIndex)
```

Comment:
The same as with `evaluate` method. Someone could use it directly bypassing lock.